SBAT.md

```diff
@@ -151,9 +153,9 @@ the UEFI revocation variable.
 If this same Vendor C has a similar event after the global number is
 incremented, they would again set their product-specific or version-specific
 number to 1. If they have a second event on with the same component, they would
-set their product or version-specific number to 2.
+set their product- or version-specific number to 2.
```

Comment:
This doesn't really make anything clearer to me; the dangling '-' is distracting.  If you're going to do this, I prefer just going ahead and doing "product-specific or version-specific".

Reply:
sure, that works too